{"uid":"2256aa877e726769","name":"test_can_create_connection[Test for Encointer, url: wss://encointer.api.onfinality.io/public-ws]","fullName":"tests.test_nodes_availability#test_can_create_connection","historyId":"8737cc376b759fe084df17567c102feb","time":{"start":1671019579929,"stop":1671019640043,"duration":60114},"status":"broken","statusMessage":"websocket._exceptions.WebSocketConnectionClosedException: Connection to remote host was lost.","statusTrace":"data = {'name': 'Encointer', 'url': 'wss://encointer.api.onfinality.io/public-ws'}\n\n    @pytest.mark.parametrize(\"data\", collect_nodes_for_chains(chains), ids=task_ids)\n    def test_can_create_connection(data):\n    \n>       connectin = create_connection_by_url(data[\"url\"])\n\ntests/test_nodes_availability.py:29: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nscripts/utils/substrate_interface.py:30: in create_connection_by_url\n    substrate = SubstrateInterface(\n../../../../.local/lib/python3.10/site-packages/substrateinterface/base.py:627: in __init__\n    self.connect_websocket()\n../../../../.local/lib/python3.10/site-packages/substrateinterface/base.py:665: in connect_websocket\n    self.websocket = create_connection(\n../../../../.local/lib/python3.10/site-packages/websocket/_core.py:608: in create_connection\n    websock.connect(url, **options)\n../../../../.local/lib/python3.10/site-packages/websocket/_core.py:253: in connect\n    self.handshake_response = handshake(self.sock, url, *addrs, **options)\n../../../../.local/lib/python3.10/site-packages/websocket/_handshake.py:57: in handshake\n    status, resp = _get_resp_headers(sock)\n../../../../.local/lib/python3.10/site-packages/websocket/_handshake.py:145: in _get_resp_headers\n    status, resp_headers, status_message = read_headers(sock)\n../../../../.local/lib/python3.10/site-packages/websocket/_http.py:312: in read_headers\n    line = recv_line(sock)\n../../../../.local/lib/python3.10/site-packages/websocket/_socket.py:131: in recv_line\n    c = recv(sock, 1)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nsock = <ssl.SSLSocket [closed] fd=-1, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6>\nbufsize = 1\n\n    def recv(sock, bufsize):\n        if not sock:\n            raise WebSocketConnectionClosedException(\"socket is already closed.\")\n    \n        def _recv():\n            try:\n                return sock.recv(bufsize)\n            except SSLWantReadError:\n                pass\n            except socket.error as exc:\n                error_code = extract_error_code(exc)\n                if error_code != errno.EAGAIN and error_code != errno.EWOULDBLOCK:\n                    raise\n    \n            sel = selectors.DefaultSelector()\n            sel.register(sock, selectors.EVENT_READ)\n    \n            r = sel.select(sock.gettimeout())\n            sel.close()\n    \n            if r:\n                return sock.recv(bufsize)\n    \n        try:\n            if sock.gettimeout() == 0:\n                bytes_ = sock.recv(bufsize)\n            else:\n                bytes_ = _recv()\n        except TimeoutError:\n            raise WebSocketTimeoutException(\"Connection timed out\")\n        except socket.timeout as e:\n            message = extract_err_message(e)\n            raise WebSocketTimeoutException(message)\n        except SSLError as e:\n            message = extract_err_message(e)\n            if isinstance(message, str) and 'timed out' in message:\n                raise WebSocketTimeoutException(message)\n            else:\n                raise\n    \n        if not bytes_:\n>           raise WebSocketConnectionClosedException(\n                \"Connection to remote host was lost.\")\nE           websocket._exceptions.WebSocketConnectionClosedException: Connection to remote host was lost.\n\n../../../../.local/lib/python3.10/site-packages/websocket/_socket.py:122: WebSocketConnectionClosedException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"broken","statusMessage":"websocket._exceptions.WebSocketConnectionClosedException: Connection to remote host was lost.","statusTrace":"data = {'name': 'Encointer', 'url': 'wss://encointer.api.onfinality.io/public-ws'}\n\n    @pytest.mark.parametrize(\"data\", collect_nodes_for_chains(chains), ids=task_ids)\n    def test_can_create_connection(data):\n    \n>       connectin = create_connection_by_url(data[\"url\"])\n\ntests/test_nodes_availability.py:29: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nscripts/utils/substrate_interface.py:30: in create_connection_by_url\n    substrate = SubstrateInterface(\n../../../../.local/lib/python3.10/site-packages/substrateinterface/base.py:627: in __init__\n    self.connect_websocket()\n../../../../.local/lib/python3.10/site-packages/substrateinterface/base.py:665: in connect_websocket\n    self.websocket = create_connection(\n../../../../.local/lib/python3.10/site-packages/websocket/_core.py:608: in create_connection\n    websock.connect(url, **options)\n../../../../.local/lib/python3.10/site-packages/websocket/_core.py:253: in connect\n    self.handshake_response = handshake(self.sock, url, *addrs, **options)\n../../../../.local/lib/python3.10/site-packages/websocket/_handshake.py:57: in handshake\n    status, resp = _get_resp_headers(sock)\n../../../../.local/lib/python3.10/site-packages/websocket/_handshake.py:145: in _get_resp_headers\n    status, resp_headers, status_message = read_headers(sock)\n../../../../.local/lib/python3.10/site-packages/websocket/_http.py:312: in read_headers\n    line = recv_line(sock)\n../../../../.local/lib/python3.10/site-packages/websocket/_socket.py:131: in recv_line\n    c = recv(sock, 1)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nsock = <ssl.SSLSocket [closed] fd=-1, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6>\nbufsize = 1\n\n    def recv(sock, bufsize):\n        if not sock:\n            raise WebSocketConnectionClosedException(\"socket is already closed.\")\n    \n        def _recv():\n            try:\n                return sock.recv(bufsize)\n            except SSLWantReadError:\n                pass\n            except socket.error as exc:\n                error_code = extract_error_code(exc)\n                if error_code != errno.EAGAIN and error_code != errno.EWOULDBLOCK:\n                    raise\n    \n            sel = selectors.DefaultSelector()\n            sel.register(sock, selectors.EVENT_READ)\n    \n            r = sel.select(sock.gettimeout())\n            sel.close()\n    \n            if r:\n                return sock.recv(bufsize)\n    \n        try:\n            if sock.gettimeout() == 0:\n                bytes_ = sock.recv(bufsize)\n            else:\n                bytes_ = _recv()\n        except TimeoutError:\n            raise WebSocketTimeoutException(\"Connection timed out\")\n        except socket.timeout as e:\n            message = extract_err_message(e)\n            raise WebSocketTimeoutException(message)\n        except SSLError as e:\n            message = extract_err_message(e)\n            if isinstance(message, str) and 'timed out' in message:\n                raise WebSocketTimeoutException(message)\n            else:\n                raise\n    \n        if not bytes_:\n>           raise WebSocketConnectionClosedException(\n                \"Connection to remote host was lost.\")\nE           websocket._exceptions.WebSocketConnectionClosedException: Connection to remote host was lost.\n\n../../../../.local/lib/python3.10/site-packages/websocket/_socket.py:122: WebSocketConnectionClosedException","steps":[],"attachments":[{"uid":"897d8a8a28ea128a","name":"stdout","source":"897d8a8a28ea128a.txt","type":"text/plain","size":260}],"parameters":[],"stepsCount":0,"attachmentsCount":1,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[],"labels":[{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_nodes_availability"},{"name":"host","value":"fv-az936-570"},{"name":"thread","value":"1747-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_nodes_availability"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"data","value":"{'url': 'wss://encointer.api.onfinality.io/public-ws', 'name': 'Encointer'}"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"d042a5cca7a84af6","status":"broken","statusDetails":"websocket._exceptions.WebSocketConnectionClosedException: Connection to remote host was lost.","time":{"start":1671019509404,"stop":1671019569811,"duration":60407}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":62,"skipped":0,"passed":290,"unknown":0,"total":352},"items":[{"uid":"1b779c6b3553cd56","reportUrl":"https://nova-wallet.github.io/test-runner/2082//#testresult/1b779c6b3553cd56","status":"broken","statusDetails":"websocket._exceptions.WebSocketConnectionClosedException: Connection to remote host was lost.","time":{"start":1671005156910,"stop":1671005217018,"duration":60108}},{"uid":"9df8a560402f4b44","reportUrl":"https://nova-wallet.github.io/test-runner/2081//#testresult/9df8a560402f4b44","status":"broken","statusDetails":"websocket._exceptions.WebSocketConnectionClosedException: Connection to remote host was lost.","time":{"start":1670990742946,"stop":1670990802782,"duration":59836}},{"uid":"9565517d3521fb90","reportUrl":"https://nova-wallet.github.io/test-runner/2080//#testresult/9565517d3521fb90","status":"broken","statusDetails":"websocket._exceptions.WebSocketConnectionClosedException: Connection to remote host was lost.","time":{"start":1670976965151,"stop":1670977024810,"duration":59659}},{"uid":"3a3547cfc7b383df","reportUrl":"https://nova-wallet.github.io/test-runner/2079//#testresult/3a3547cfc7b383df","status":"broken","statusDetails":"websocket._exceptions.WebSocketConnectionClosedException: Connection to remote host was lost.","time":{"start":1670961948123,"stop":1670962007791,"duration":59668}},{"uid":"455447878af4d3a5","reportUrl":"https://nova-wallet.github.io/test-runner/2078//#testresult/455447878af4d3a5","status":"broken","statusDetails":"websocket._exceptions.WebSocketConnectionClosedException: Connection to remote host was lost.","time":{"start":1670947555899,"stop":1670947615810,"duration":59911}},{"uid":"da2dff79375ed9ea","reportUrl":"https://nova-wallet.github.io/test-runner/2077//#testresult/da2dff79375ed9ea","status":"broken","statusDetails":"websocket._exceptions.WebSocketConnectionClosedException: Connection to remote host was lost.","time":{"start":1670933197901,"stop":1670933257790,"duration":59889}},{"uid":"a02e66a885f3e031","reportUrl":"https://nova-wallet.github.io/test-runner/2076//#testresult/a02e66a885f3e031","status":"broken","statusDetails":"websocket._exceptions.WebSocketConnectionClosedException: Connection to remote host was lost.","time":{"start":1670918761802,"stop":1670918821677,"duration":59875}},{"uid":"ce056ff8f5fbbdfb","reportUrl":"https://nova-wallet.github.io/test-runner/2075//#testresult/ce056ff8f5fbbdfb","status":"passed","time":{"start":1670904274250,"stop":1670904274932,"duration":682}},{"uid":"947308ed12306a2b","reportUrl":"https://nova-wallet.github.io/test-runner/2074//#testresult/947308ed12306a2b","status":"passed","time":{"start":1670890455468,"stop":1670890456084,"duration":616}},{"uid":"83528b8f3fae1099","reportUrl":"https://nova-wallet.github.io/test-runner/2073//#testresult/83528b8f3fae1099","status":"passed","time":{"start":1670875443372,"stop":1670875444038,"duration":666}},{"uid":"be7323841abb1917","reportUrl":"https://nova-wallet.github.io/test-runner/2072//#testresult/be7323841abb1917","status":"passed","time":{"start":1670861070067,"stop":1670861070580,"duration":513}},{"uid":"2552dbfbef3702a8","reportUrl":"https://nova-wallet.github.io/test-runner/2071//#testresult/2552dbfbef3702a8","status":"passed","time":{"start":1670846740139,"stop":1670846740681,"duration":542}},{"uid":"e4894829aef9eb98","reportUrl":"https://nova-wallet.github.io/test-runner/2070//#testresult/e4894829aef9eb98","status":"passed","time":{"start":1670832314356,"stop":1670832315068,"duration":712}},{"uid":"8a534cc20bca8d95","reportUrl":"https://nova-wallet.github.io/test-runner/2069//#testresult/8a534cc20bca8d95","status":"passed","time":{"start":1670817894711,"stop":1670817895309,"duration":598}},{"uid":"442ce92ca096f2c5","reportUrl":"https://nova-wallet.github.io/test-runner/2068//#testresult/442ce92ca096f2c5","status":"passed","time":{"start":1670804066882,"stop":1670804067471,"duration":589}},{"uid":"1438e15c645dd243","reportUrl":"https://nova-wallet.github.io/test-runner/2067//#testresult/1438e15c645dd243","status":"passed","time":{"start":1670789067765,"stop":1670789068475,"duration":710}},{"uid":"842e0a59acc2735b","reportUrl":"https://nova-wallet.github.io/test-runner/2066//#testresult/842e0a59acc2735b","status":"passed","time":{"start":1670774663355,"stop":1670774664067,"duration":712}},{"uid":"c402be7f933555a5","reportUrl":"https://nova-wallet.github.io/test-runner/2065//#testresult/c402be7f933555a5","status":"passed","time":{"start":1670760326546,"stop":1670760327061,"duration":515}},{"uid":"71c632d4fad8a766","reportUrl":"https://nova-wallet.github.io/test-runner/2064//#testresult/71c632d4fad8a766","status":"passed","time":{"start":1670745957475,"stop":1670745958154,"duration":679}},{"uid":"45cdca4500bf4c25","reportUrl":"https://nova-wallet.github.io/test-runner/2063//#testresult/45cdca4500bf4c25","status":"passed","time":{"start":1670731563604,"stop":1670731564325,"duration":721}}]},"tags":[]},"source":"2256aa877e726769.json","parameterValues":["{'url': 'wss://encointer.api.onfinality.io/public-ws', 'name': 'Encointer'}"]}